src/components/default-page-template.js

```diff
@@ -172,6 +172,7 @@ function DefaultPageTemplate({ children, fullScreen, headerActions, alerts }) {
           ))}
         </div>
       </footer>
+      <CookieConsentDrawer cookieName={clientConfig.consentCookieName} />
```

Comment:
There's no need to pass down the client config as a prop. The `CookieConsentDrawer` has access to the client config itself.

Reply:
Hmm, how?

Reply:
`useService(ClientConfig)`?

Reply:
Fixed.